code/svb-docker/python/svb_web/core/templates/internal/edit_customer.html

```diff
@@ -0,0 +1,31 @@
+{% extends "internal/base.html" %}
+
+{% block content %}
+<div class="container p-5">
+    <h1>Edit Customer</h1>
+    <div class="row">
+        <div class="col-sm">
+            <h1>{{form_title}}</h1>
+            <div class="mt-2 mb-2 {{existing_id_banner_display}}">

```

Comment:
NP: consistent styling:
```suggestion
            <div class="mt-2 mb-2 {{ existing_id_banner_display }}">

```